### ISSUE TITLE
boundary error when calling coveralls in .travis.yml

### ISSUE THREAD

**Issue by stephlocke** [jimhester/covr#81]
Bumped my run now that the version dependencies were relaxed. Got an issue with the boundary.

See [optiRum log](https://travis-ci.org/stephlocke/optiRum/builds) but main error transcribed:
```
$ Rscript -e 'library(covr);coveralls()'
Loading required package: methods
Error in eval(expr, envir, enclos) : object 'boundary' not found
Calls: coveralls ... escape -> <Anonymous> -> lapply -> FUN -> eval -> eval
Execution halted
```

**Comment by jimhester** [comment]
Should be fixed with 6c6057d8689325fb41f8e4d6a1fd023c387253c6 a29947104520716bea8cc4e93d5976d7945a7bbd, thank you for opening the issue!

**Comment by stephlocke** [comment]
Cheers :+1:
